### PR TITLE
man-db: import from core

### PR DESCRIPTION
**Pull request by bw-yuhogg**
After this is merged, I also intend to upgrade both `libpipeline` and `man-db` to newer versions.

Also note that I have to make a tap migration for these in core.

### CODE REVIEW

File: Formula/libpipeline.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Libpipeline < Formula
+  desc "Library for manipulating pipelines of subprocesses"
+  homepage "http://libpipeline.nongnu.org/"
+  url "https://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.3.1.tar.gz"
+  sha256 "5cad1b446f135ec3800d32c8c951a1114f4c438609a4c52b262c30301bc8e692"
+  # tag "linuxbrew"
```
Comment (sjackman) [inline Formula/libpipeline.rb:6]: We can remove these tags now.

**Comment by sjackman** [comment]
Looks good to me. Thanks, Bob. It'd be great if you have time to fix `A 'test do' test block should be added`

File: Formula/libpipeline.rb (hunk 1 above)
Comment (bw-yuhogg) [reply Formula/libpipeline.rb:6]: I kept them because they're ubiquitous in the xorg tap; would you like me to bulk remove them from there too?

**Comment by bw-yuhogg** [comment]
The failures were just that these 2 formulae conflict with existing core formulae. Not an issue.

**Pull request closed without merging**